src/Components/Components/src/Microsoft.AspNetCore.Components.csproj

```diff
@@ -1,12 +1,11 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Components feature for ASP.NET Core.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsShippingPackage>true</IsShippingPackage>
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
-    <RazorLangVersion>3.0</RazorLangVersion>
```

Comment:
Do we really want to undo this? Wouldn't this just get put back in the future if we decide to use a `.razor` file here?

Reply:
Sure, I thought we’d just put it back if we have a reason to. It doesn’t produce any visible difference to the compiled package does it? As in, there’s nothing breaking or even visible about putting it back later.

I don’t mind. I’m happy to put it back if there’s a preference for that.

Reply:
I don't really take issue either way. It just seems like homework.

Reply:
OK, since you don't mind I'll leave it as-is. We can trivially put it back if there's reason to later.